prbt_hardware_support/srv/SendBrakeTestResult.srv

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Result of brake test to be sent
+uint16 performed
+uint16 result
```

Comment:
Was is the meaning of this value? Where is it defined?

Reply:
Simplified and documented SendBrakeTestResult service. -> Done